app/controllers/audience_controller.rb

```diff
@@ -16,8 +16,16 @@ def index
   def export
     authorize :audience
 
-    audience_csv = Exports::AudienceExportService.new(current_seller).perform
-    send_data audience_csv, type: "text/csv"
+    options = params.required(:options)
+                 .permit(:followers, :customers, :affiliates)
+                 .to_h
+                 .transform_keys(&:to_s)
+
+    options.transform_values! { |v| ActiveModel::Type::Boolean.new.cast(v) }
+
+    Exports::AudienceExportWorker.perform_async(current_seller.id, (impersonating_user || current_seller).id, { **options })
+
+    render json: { success: true }
```

Comment:
```suggestion
    head :ok
```

since we don't actually have anything meaningful to return